ES/ESmy018.xml

```diff
@@ -47,133 +47,138 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
    <locus from="13ra" to="14va"/>
    <title type="complete" ref="LIT3613Miracle"/>
    <note>7</note>
 </msItem>
 
-<msItem xml:id="ms_i1.9">
+<msItem xml:id="ms_i1.2.8">
  <locus from="14va" to="15rb"/>
  <title type="complete" ref="LIT3597Miracle"/>
  <note>8</note>
 </msItem>
 
-<msItem xml:id="ms_i1.10">
+<msItem xml:id="ms_i1.2.9">
  <locus from="15rb" to="16ra"/>
  <title type="complete" ref="LIT3925Miracle"/>
  <note>9</note>
 </msItem>
 
-<msItem xml:id="ms_i1.11">
+<msItem xml:id="ms_i1.2.10">
  <locus from="16ra" to="17va"/>
  <title type="complete" ref="LIT3724Miracle"/>
  <note>numbered 10</note>
 </msItem>
 
-<msItem xml:id="ms_i1.12">
+<msItem xml:id="ms_i1.2.11">
  <locus from="17va" to="19vb"/>
  <title type="complete">A miracle that involves the sycomonre of Dabra Mǝṭmāq, similar to
  <ref type="work" corresp="LIT3664Miracle"/>.</title>
  <note>11</note>
 </msItem>
 
-<msItem xml:id="ms_i1.13">
+<msItem xml:id="ms_i1.2.12">
  <locus from="19vb" to="20vb"/>
  <title type="complete" corresp="LIT3590Miracle">The name of the pious monastic scribe who
    wrote the name of St Mary in golden ink is not explicetly mentioned (otherwise Damianus from Philippi).</title>
  <note>12</note>
 </msItem>
 
-<msItem xml:id="ms_i1.14">
+<msItem xml:id="ms_i1.2.13">
  <locus from="20vb" to="22rb"/>
  <title type="complete" corresp="LIT3725Miracle"></title>
  <note>numbered 13</note>
 </msItem>
 
-<msItem xml:id="ms_i1.15">
+<msItem xml:id="ms_i1.2.14">
  <locus from="22vb" to="23vb"/>
  <title type="complete" corresp="LIT3869Miracle"></title>
  <note>numbered 14</note>
 </msItem>
 
-<msItem xml:id="ms_i1.16">
+<msItem xml:id="ms_i1.2.15">
  <locus from="23vb" to="25rb"/>
  <title type="complete" corresp="LIT3640Miracle"></title>
  <note>numbered 15</note>
 </msItem>
 
-<msItem xml:id="ms_i1.17">
+<msItem xml:id="ms_i1.2.16">
  <locus from="25rb" to="28ra"/>
  <title type="complete" corresp="LIT3615Miracle"></title>
  <note>numbered 16</note>
 </msItem>
 
-<msItem xml:id="ms_i1.18">
+<msItem xml:id="ms_i1.2.17">
  <locus from="27ra" to="28vb"/>
  <title type="complete" corresp="LIT3596Miracle">The name of the protagonist is not mentioned (elsewhere
    he is Zechariah of Rome).</title>
  <note>numbered 17</note>
 </msItem>
 
-<msItem xml:id="ms_i1.19">
+<msItem xml:id="ms_i1.2.18">
  <locus from="28vb" to="30va"/>
  <title type="complete" corresp="LIT3920Miracle">The name of the protagonist is not mentioned (elsewhere
    he is Gerdan), some details are different.</title>
  <note>numbered 18</note>
 </msItem>
 
-<msItem xml:id="ms_i1.20">
+<msItem xml:id="ms_i1.2.19">
  <locus from="30va" to="32ra"/>
  <title type="complete" corresp="LIT5312MiracleAnaphora"></title>
  <note>numbered 19</note>
 </msItem>
 
-<msItem xml:id="ms_i1.21">
+<msItem xml:id="ms_i1.2.20">
  <locus from="32ra" to="33ra"/>
  <title type="complete" corresp="LIT3692Miracle"></title>
  <note>numbered 20</note>
 </msItem>
 
-<msItem xml:id="ms_i1.22">
+<msItem xml:id="ms_i1.21">
```

Comment:
change ID to
ms_i1.2.21
here and for 1.32 --> 1.2.32

Reply:
Ok